src/DotNetLightning.Core/Channel/Commitments.fs

```diff
@@ -110,7 +110,7 @@ type RemoteNextCommitInfo =
             (fun commitmentPubKey remoteNextCommitInfo ->
                 match remoteNextCommitInfo with
                 | Waiting _ -> remoteNextCommitInfo
-                | Revoked commitmentPubKecommitmentPubKey -> Revoked commitmentPubKey)
+                | Revoked _commitmentPubKecommitmentPubKey -> Revoked commitmentPubKey)
```

Comment:
@joemphilips note on this line change above ^ not only the variable name is weird (we should fix it)... is it also a bug? I mean maybe the result should be `Revoked commitmentPubKecommitmentPubKey` (or whatever new var name we choose) instead of `Revoked commitmentPubKey`; let me know what you think

Reply:
It is not a bug but you are right the variable name is weird. `commitmentPubKecommitmentPubKey` should be just `_` as in the case for in the `Waiting_`

Reply:
cool thanks I'll rename

Reply:
I don't know why I ended up with this nonsense variable name. Probably  the editor fooled me.